hbase-client/src/main/java/org/apache/hadoop/hbase/client/TableDescriptorBuilder.java

```diff
@@ -234,6 +242,7 @@ public class TableDescriptorBuilder {
     DEFAULT_VALUES.put(DURABILITY, DEFAULT_DURABLITY.name()); // use the enum name
     DEFAULT_VALUES.put(REGION_REPLICATION, String.valueOf(DEFAULT_REGION_REPLICATION));
     DEFAULT_VALUES.put(PRIORITY, String.valueOf(DEFAULT_PRIORITY));
+    DEFAULT_VALUES.put(ERASURE_CODING_POLICY, String.valueOf(DEFAULT_ERASURE_CODING_POLICY));
```

Comment:
This will lead to a "null" string? Is this the expected behavior?

Reply:
I don't think this is a problem, because it's only used in toString in order to avoid printing default values. It's effectively unused, because we don't keep null values in our htd hashmap so nothing would ever match this. If someone tried to set a value of `null` (string), the input validation would fail. 

I can't remember why I added this, except that it looked like defaults should be registered here. But now that you point it out, I should probably remove it. I can do that in a follow-up jira since this is just a backport and the same code exists in master/branch-3.

Reply:
To answer your question, this is as designed. But as I think through it now, it's unnecessary and confusing, though not a problem.

Reply:
https://issues.apache.org/jira/browse/HBASE-28334

Reply:
In the above jira, we realized that it's necessary to set something into the DEFAULT_VALUES map for this. Otherwise, it gets printed as METADATA in the toString representation. We added a comment to note that, and the value does not matter.

Any other concerns on this PR @Apache9? Thanks for looking